isa/rv32ud/fcvt_w.S

```diff
@@ -4,4 +4,4 @@
 #undef RVTEST_RV64UF
 #define RVTEST_RV64UF RVTEST_RV32UF
```

Comment:
```suggestion
#define RVTEST_RV64UD RVTEST_RV32UD
```
?

Reply:
All of the files in rv32ud have the same #define, if that is incorrect, they should all be changed. I didn't know if that was a mistake or not so I didn't change it.

Reply:
Hello, I think the #define RVTEST_RV64UF RVTEST_RV32UF part is not mistake, because the original define enables the floating point unit, but if you define it as RVTEST_RV64UD , the floating unit would not be enabled. The macro definition is in riscv_test.h, but the include is likely a mistake !